R/testing.R

```diff
@@ -146,6 +146,11 @@ hook_state_create <- function(tempdir,
   withr::local_dir(tempdir)
   files <- fs::path_rel(path_candidate_temp, tempdir)
   # https://stat.ethz.ch/pipermail/r-devel/2018-February/075507.html
+
+  # quote any individual filenames with spaces so the shell identifies them
+  # each as a single term
+  files <- ifelse( grepl(' ', files), shQuote(files), files)
```

Comment:
Can we `shQuote()` even when the files don't contain a space? Then, we can spare the conditional.